ontolib-core/src/main/java/com/github/phenomics/ontolib/ontology/data/ImmutableOntology.java

```diff
@@ -1,9 +1,6 @@
 package com.github.phenomics.ontolib.ontology.data;
 
-import java.util.Collection;
```

Comment:
Please no star imports but explicit ones. I guess you can easily adjust your IDE to do this.

Reply:
Also, I forgot: please add note to Changelog.